lib/services.identity.js

```diff
@@ -516,11 +516,6 @@ api._createIdentity = function(options, req, callback) {
     }
     // result details
     var details = {identity: results.createIdentity};
-    // schedule identity created event
-    bedrock.events.emitLater({
-      type: 'bedrock.Identity.created',
```

Comment:
It looks like bedrock-mail doesn't understand how to use the emit paramaters that postInsert is using -- postInsert takes comma seperated paramaters https://github.com/digitalbazaar/bedrock-identity/blob/master/lib/identity.js#L114, but the IDP's event here uses the old way of using one event object with a details value. So, this fails when we switch over to postInsert because the mail template can't find any details objects.